components/stepper/stories/stepper.stories.js

```diff
@@ -76,133 +81,53 @@ export default {
-	})}>
-		${Template({
-			...args,
-			isFocused: true,
-		})}
-		${Template({
-			...args,
-			isKeyboardFocused: true,
-		})}
-		${Template({
-			...args,
-			isInvalid: true,
-		})}
-		${Template({
-			...args,
-			isInvalid: true,
-			isFocused: true,
-		})}
-		${Template({
-			...args,
-			isInvalid: true,
-			isKeyboardFocused: true,
-		})}
-		${Template({
-			...args,
-			isDisabled: true,
-		})}
-		${Template({
-			...args,
-			isQuiet: true,
-		})}
-		${Template({
-			...args,
-			isQuiet: true,
-			isFocused: true,
-		})}
-		${Template({
-			...args,
-			isQuiet: true,
-			isKeyboardFocused: true,
-		})}
-		${Template({
-			...args,
-			isQuiet: true,
-			isInvalid: true,
-		})}
-		${Template({
-			...args,
-			isQuiet: true,
-			isInvalid: true,
-			isFocused: true,
-		})}
-		${Template({
-			...args,
-			isQuiet: true,
-			isInvalid: true,
-			isKeyboardFocused: true,
-		})}
-		${Template({
-			...args,
-			isQuiet: true,
-			isDisabled: true,
-		})}
-		${Template({
-			...args,
-			hideStepper: true,
-		})}
-		${Template({
-			...args,
-			hideStepper: true,
-			isFocused: true,
-		})}
-		${Template({
-			...args,
-			hideStepper: true,
-			isKeyboardFocused: true,
-		})}
-		${Template({
-			...args,
-			hideStepper: true,
-			isInvalid: true,
-		})}
-		${Template({
-			...args,
-			hideStepper: true,
-			isInvalid: true,
-			isFocused: true,
-		})}
-		${Template({
-			...args,
-			hideStepper: true,
-			isInvalid: true,
-			isKeyboardFocused: true,
-		})}
-		${Template({
-			...args,
-			hideStepper: true,
-			isDisabled: true,
-		})}
-	</div>
-`;
 
-export const Default = Variants.bind({});
+export const Default = StepperGroup.bind({});
 Default.args = {};
+Default.tags = ["!autodocs"];
+
+export const Sizing = SizingTemplate.bind({});
+Sizing.tags = ["autodocs", "!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const DefaultStates = DefaultTemplate.bind({});
+DefaultStates.args = {};
+DefaultStates.tags = ["autodocs", "!dev"];
+DefaultStates.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+DefaultStates.storyName = "Default";
+
+export const DisabledStates = DisabledTemplate.bind({});
+DisabledStates.args = {};
+DisabledStates.tags = ["autodocs", "!dev"];
+DisabledStates.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+DisabledStates.storyName = "Disabled";
+
+export const QuietStates = QuietTemplate.bind({});
+QuietStates.args = {};
+QuietStates.tags = ["autodocs", "!dev"];
+QuietStates.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+QuietStates.storyName = "Quiet";
+
+export const HideStepper = Template.bind({});
+HideStepper.args = {
+	hideStepper: true,
+};
+HideStepper.tags = ["autodocs", "!dev"];
 
 // ********* VRT ONLY ********* //
-export const WithForcedColors = Variants.bind({});
-WithForcedColors.tags = ["!autodocs", "!dev"];
+export const WithForcedColors = Default.bind({});
+WithForcedColors.tags = ["!autodocs", "!dev", "test"];
```

Comment:
```suggestion
WithForcedColors.tags = ["!autodocs", "!dev"];
```

No need to add back the "test" tag; it doesn't do anything at this time.